ivadomed/training.py

```diff
@@ -405,10 +405,10 @@ def get_scheduler(params, optimizer, num_epochs=0):
         scheduler = optim.lr_scheduler.CyclicLR(optimizer, **params, mode="triangular2", cycle_momentum=False)
         step_scheduler_batch = True
     else:
-        print(
+        raise ValueError(
             "Unknown LR Scheduler name, please choose between 'CosineAnnealingLR', 'CosineAnnealingWarmRestarts',"
             "or 'CyclicLR'")
```

Comment:
```suggestion
            "Unknown LR Scheduler name, please choose between 'CosineAnnealingLR', 'CosineAnnealingWarmRestarts',"
            " or 'CyclicLR'")
```

Reply:
I would recommend printing the unknown LR sch. name

Reply:
done in 
a3e884d